app/models/content_item.rb

```diff
@@ -0,0 +1,53 @@
+class ContentItem
+  attr_reader :content_item, :body, :image, :description, :document_type, :title, :base_path, :locale
+
+  def initialize(content_store_response)
+    @content_item = content_store_response
+    @body = content_item.dig("details", "body")
+    @image = content_item.dig("details", "image")
+    @description = content_item["description"]
+    @document_type = content_item["document_type"]
+    @title = content_item["title"]
+    @base_path = content_item["base_path"]
+    @locale = content_item["locale"]
+  end
+
+  def available_translations
+    translations = content_item["links"]["available_translations"] || []
+
+    mapped_locales(sorted_locales(translations))
+  end
+
+  def sorted_locales(translations)
+    translations.sort_by { |t| t["locale"] == I18n.default_locale.to_s ? "" : t["locale"] }
+  end
+
+  def show_default_breadcrumbs?
+    true
+  end
+
+  def parsed_content_item
```

Comment:
What is this expected to be used for?

Reply:
Removed (not necessary for this small change)